### PR TITLE
Feature/#21 post creation

### PR DESCRIPTION
**Pull request by ArtemDymytrov**
1.Create modal window with post form creation.
### **Desctop**
image

 **Mobile**   |   **Empty field**
:-------------------------:|:-------------------------:
image  |  image

**With field**             |  **Error**
:-------------------------:|:-------------------------:
image  |  image

### CODE REVIEW

File: src/components/PostCreation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+export const PostCreationModal: React.FC<{ onCrossBtnHandler: React.MouseEventHandler }> = ({ onCrossBtnHandler }) => {
+  const handleSubmit = async (post: ICreatePostRequest) => {
+    PostsService.addPost(post)
+      .then(() =>
+        toasterService.success({
+          title: 'Success',
+          content: 'Post uploaded:)',
+          duration: 3000,
+        }),
```
Comment (Chubatik) [inline src/components/PostCreation/index.tsx:22]: duration already equal to 3 seconds, no need to point again

Comment (ArtemDymytrov) [reply src/components/PostCreation/index.tsx:22]: Okkay thanks, i will change it

File: src/components/PostCreation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              + Create Post
+            </Button>
+          </div>
+        </form>
+      </Modal>
+    </React.Fragment>
+  )
+}
+
+export default PostCreationModal
```
Comment (Chubatik) [inline src/components/PostCreation/index.tsx:96]: Why do you export component again ?

Comment (ArtemDymytrov) [reply src/components/PostCreation/index.tsx:96]: FIxed

**Review by Chubatik — APPROVED** [review]

**Review by YanaGushlevskaya — APPROVED** [review]

**Review by vihopec — APPROVED** [review]

**Pull request merged**